supervisor/coresys.py

```diff
@@ -88,6 +89,11 @@ def __init__(self):
         self._jobs: Optional[JobManager] = None
         self._security: Optional[Security] = None
 
+        # Set default header for aiohttp
+        self._websession._default_headers = MappingProxyType(
+            {aiohttp.hdrs.USER_AGENT: SERVER_SOFTWARE}
```

Comment:
I think that can be difficult for ingress? Maybe we just set `ClientSession(..., headers={USER_AGENT})` or do you think it's not an issue if we overwrite that?

Reply:
It should not be an issue (we already do this on the core side), but I can test ingress later today

Reply:
Tested ingress with FileEditor, VSCode, Node-RED, and my 2 test add-ons, no issues with any of them :+1: